whitepaper/whitepaper.md

```diff
@@ -124,12 +133,248 @@ Blockchain technologies are growing in usage, but fragmentation is a big problem
 
 We propose a protocol and it's implementation to connect as many of them as possible in an attempt to solve the fragmentation problem by creating a heterogeneous system architecture <sup>[1](#7-references)</sup>.
 
+
+
+There are two inherent problems<sup>[a](#12-footnotes)</sup> that have to be solved when connecting different blockchains: 
+* How to retrieve the
```

Comment:
It could be possible to view the ledger information, but I have little concerns about expected function. Cactus will not provide any retrieval of ledger information. We need to clarification about the functions in this topic. You may know about the project Explorer for generic use  is proposed at TSC. If we can collaborate with them, it can be beneficial.

Reply:
> Cactus will not provide any retrieval of ledger information.

I would say that providing proofs about the ledger state is what one could call as "retrieval of ledger information"
Would also say that while Cactus itself may not be doing the ledger querying, the plugins certainly are. It's an interesting question how should this be phrased. If a Cactus plugin is doing something, can we say that Cactus is doing that thing?
Technically if someone wanted to (not me, but if someone had that motivation for whatever reason), they could implement the entire HL Explorer project as a Cactus plugin and then deploy that.